parlai/crowdsourcing/tasks/model_chat/worlds.py

```diff
@@ -225,6 +225,15 @@ def parley(self):
         for idx, agent in enumerate([self.agent, self.bot]):
             if not self.chat_done:
                 acts[idx] = agent.act(timeout=self.max_resp_time)
+                if (
```

Comment:
Yeah I think this is preferable to what was here previously, but now the Turker sees `self.bot_agent_id` as the name of the bot, right? Orion implemented a fix in his eval task to just set the bot display name for the Turkers as `YOUR PARTNER` so that they don't see the name of the bot - we might end up using that across all model chat tasks

Reply:
sounds good. is the display name for bot configurable in Orion's implementation?

Reply:
I don't think it currently is, but it'd be good for it to be